compiler/parser/src/lib.rs

```diff
@@ -44,7 +43,7 @@ pub fn parse_ast<N: Network>(
     handler: Handler,
     node_builder: &NodeBuilder,
     source: &str,
-    start_pos: BytePos,
+    start_pos: u32,
```

Comment:
Should this be renamed to `absolute_start` for consistency?
Ditto for similar instances.